x/validator/types/errors.go

```diff
@@ -20,9 +20,47 @@ import (
 	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
 )
 
-var PoolIsFull = sdkerrors.Register(ModuleName, 601, "maximum number of active nodes reached")
+var (
+	PoolIsFull                               = sdkerrors.Register(ModuleName, 601, "maximum number of active nodes reached")
+	ErrProposedDisableValidatorAlreadyExists = sdkerrors.Register(ModuleName, 801, "disable validator propose already exists")
+	ErrProposedDisableValidatorDoesNotExist  = sdkerrors.Register(ModuleName, 802, "disable validator propose does not exist")
+	ErrDisabledValidatorAlreadytExists       = sdkerrors.Register(ModuleName, 803, "disabled validator already exist")
+	ErrDisabledValidatorDoesNotExist         = sdkerrors.Register(ModuleName, 804, "disabled validator does not exist")
+)
```

Comment:
Correct error codes. 8xx series is used by `dclupgrade` module. `validator` module uses 6xx series.